packages/map-template/src/components/WebComponentWrappers/Search/Search.jsx

```diff
@@ -97,6 +103,7 @@ const SearchField = forwardRef(({ placeholder, mapsindoors, results, clicked, cl
     }, [placeholder, mapsindoors, results, clicked, cleared, google, mapbox, changed]);
 
     return <mi-search ref={elementRef}
+        id-attribute='id'
```

Comment:
Was this needed, or was that part of some debugging? I can't remember.

Reply:
I think it was needed to fix the warning